package.json

```diff
@@ -13,36 +13,59 @@
-    "build_cjs": "rm -rf dist/cjs && tsc typings/es6-shim/es6-shim.d.ts src/Rx.ts src/Rx.KitchenSink.ts src/Rx.DOM.ts src/add/observable/of.ts -m commonjs --outDir dist/cjs --sourcemap --sourceRoot src --target ES5 -d --diagnostics --pretty --noImplicitAny --suppressImplicitAnyIndexErrors",
-    "build_es6": "rm -rf dist/es6 && tsc src/Rx.ts src/Rx.KitchenSink.ts src/Rx.DOM.ts src/add/observable/of.ts --outDir dist/es6 --sourceMap --sourceRoot src --target ES6 -d --diagnostics --pretty --noImplicitAny --suppressImplicitAnyIndexErrors",
+    "build_amd": "npm run clean_dist_amd && npm run copy_src_amd && npm run compile_dist_amd",
+    "build_cjs": "npm run clean_dist_cjs && npm run copy_src_cjs && npm run compile_dist_cjs",
+    "build_es6": "npm run clean_dist_es6 && npm run copy_src_es6 && npm run compile_dist_es6",
+
     "build_closure_core": "java -jar ./node_modules/google-closure-compiler/compiler.jar --js ./dist/global/Rx.umd.js --language_in ECMASCRIPT5 --create_source_map ./dist/global/Rx.umd.min.js.map --js_output_file ./dist/global/Rx.umd.min.js",
     "build_closure_kitchensink": "java -jar ./node_modules/google-closure-compiler/compiler.jar --js ./dist/global/Rx.KitchenSink.umd.js --language_in ECMASCRIPT5 --create_source_map ./dist/global/Rx.KitchenSink.umd.min.js.map --js_output_file ./dist/global/Rx.KitchenSink.umd.min.js",
-    "build_global": "rm -rf dist/global && mkdir \"dist/global\" && node tools/make-umd-bundle.js && node tools/make-system-bundle.js && npm run build_closure_core && npm run build_closure_kitchensink",
+    "build_global": "rm -rf ./dist/global && mkdir \"./dist/global\" && node tools/make-umd-bundle.js && node tools/make-system-bundle.js && npm run build_closure_core && npm run build_closure_kitchensink",
     "build_perf": "npm run build_cjs && npm run build_global && webdriver-manager update && npm run perf",
-    "build_test": "rm -rf dist/ && npm run lint && npm run build_cjs && npm run test_buildonly && npm run test_nobuild",
-    "build_cover": "rm -rf dist/ && npm run lint && npm run build_cjs && npm run test_buildonly && npm run cover",
+    "build_test": "rm -rf ./dist/ && npm run lint && npm run build_cjs && npm run test_build && npm run test_jasmine",
+    "build_cover": "rm -rf ./dist/ && npm run lint && npm run build_cjs && npm run test_build && npm run cover",
     "build_docs": "npm run build_es6 && npm run build_global && npm run tests2png && esdoc -c esdoc.json",
-    "publish_docs": "./publish_docs.sh",
+
+    "check_circular_dependencies": "madge ./dist/cjs --circular",
+
+    "clean_dist_amd": "rm -rf ./dist/amd",
+    "clean_dist_cjs": "rm -rf ./dist/cjs",
+    "clean_dist_es6": "rm -rf ./dist/es6",
+
+    "copy_src_amd": "mkdir -p ./dist/amd && cp -r ./src/* ./dist/amd/",
```

Comment:
No, I always updated script windows compatible (yes, I'm one of few windows user ;) )